web/thesauruses/haskell/lists.json

```diff
@@ -0,0 +1,386 @@
+      "not-implemented": true
+    },
+    "delete_ordered_immutable_list": {
+      "not-implemented": true
+    },
+    "name_of_unordered_immutable_list": {
+      "comment": "`a` is the type of elements of the list",
+      "code": "[a]"
+    },
+    "create_a_unordered_immutable_list": {
+      "code": "[firstElement, secondElement]"
+    },
+    "unordered_immutable_list_start_number": {
+      "code": "0"
+    },
+    "access_element_in_unordered_immutable_list": {
+      "code": "list !! index"
+    },
+    "delete_unordered_immutable_list": {
+      "not-implemented": true
+    },
+    "name_of_mutable_hashed_list": {
+      "not-implemented": true
+    },
+    "create_a_mutable_hashed_list": {
+      "not-implemented": true
+    },
+    "insert_element_to_mutable_hashed_list": {
+      "not-implemented": true
+    },
+    "erase_element_from_mutable_hashed_list": {
+      "not-implemented": true
+    },
+    "delete_mutable_hashed_list": {
+      "not-implemented": true
+    },
+    "name_of_immutable_hashed_list": {
+      "code": "Set"
+    },
+    "create_a_immutable_hashed_list": {
+      "code": "Set.empty\nSet.fromList list\nSet.fromAscList ascendingList\nSet.fromDescList descendingList\nSet.fromDistinctAscList list\nSet.fromDistinctDescList list"
+    },
+    "insert_element_to_immutable_hashed_list": {
+      "code": "insert setToInsertTo elementToInsert"
+    },
+    "erase_element_from_immutable_hashed_list": {
+      "code": "delete setToDeleteFrom elementToDelete"
+    },
+    "delete_immutable_hashed_list": {
+      "not-implemented": true
+    },
+    "create_a_mutable_set": {
+      "not-implemented": true
+    },
+    "get_key_from_mutable_set": {
+      "not-implemented": true
+    },
+    "get_value_from_mutable_set": {
+      "not-implemented": true
+    },
+    "get_all_keys_from_mutable_set": {
+      "not-implemented": true
+    },
+    "get_all_values_from_mutable_set": {
+      "not-implemented": true
+    },
+    "swap_key_and_value_in_mutable_set": {
+      "not-implemented": true
+    },
+    "delete_mutable_set": {
+      "not-implemented": true
+    },
+    "create_a_immutable_set": {
+      "code": "import qualified Data.Map as Map\nMap.empty\nMap.singleton key value -- Map containing a single entry\nMap.fromSet generateValueFromKey set\nMap.fromList [(key1, value1), (key2, value2)]",
+      "comment": "choose `Data.Map.Strict` or `Data.Map.Lazy` depending on your requirements"
+    },
+    "get_key_from_immutable_set": {
+      "not-implemented": true
+    },
+    "get_value_from_immutable_set": {
+      "code": "map ! key"
+    },
+    "get_all_keys_from_immutable_set": {
+      "code": "keys map"
+    },
+    "get_all_values_from_immutable_set": {
+      "code": "elems map"
+    },
+    "swap_key_and_value_in_immutable_set": {
+      "not-implemented": true
+    },
+    "delete_immutable_set": {
+      "not-implemented": true
+    },
+    "find_element_at_position": {
+    },
```

Comment:
What's the difference of this and `access_element_in_unordered_immutable_list` in line 51?

Reply:
I think this was me adding this before I added it to each individual list. It should probably be removed from the meta structure file. I'll remove it there, therefore you can remove it here too.